scripts/plugins/logging.py

```diff
@@ -39,8 +39,11 @@ def mask_message(message):
     """
     if message is None:
         return message
+    if BITOPS_logging_masks is None:
+        return message
```

Comment:
Is this change relevant to the original PR's topic of installing plugins?
I'd recommend splitting different changes into separated PRs in the future.

Reply:
I know. 
But there is something to be said about getting small fixes that will benefit the system in with minimal headache. 

I understand why we may not want to do that, as it means in the case of needing to work backwards, it becomes harder. 

But a whole PR for a single none check also seems like overkill

Reply:
https://fagnerbrack.com/one-pull-request-one-concern-e84a27dfe9f1

I think it's important and helps scale the collaboration. It's OK to do big overhauls in one PR when there are just a couple of folks involved doing isolated work, but when we're building a community with potential external contributors it's best to show the best practices.

A few benefits that come into mind:
- The PR having one concern is easier to review and faster to merge. See this PR is blocked by a discussion on whether to expose the parameter in the config or not. But that included extra fix could be already merged if created separately and the issue could be already closed. What if someone was waiting for that specific issue to get fixed? 
- History/Changelog - when we have a specific PR for a specific problem describing the issue and solution, - it's more organised and helps others to understand the bug-fix scope. Say someone using a fork is experiencing a problem we're fixing and they're looking for a diff. This PR doesn't provide that context as the solution is hidden behind the other feature. We're also composing Changelog for the Release notes where every PR represents specific functionality or isolated fix.
- Reverts - sometimes reverting PRs is needed. Especially if you're in the SRE world where mean time to recovery is critical. With multiple changes it's hard to revert a specific change.
- Testing. When we'll start adding unit tests with every fix or new feature these problems will pop up more with a code diff.

One concern PRs just creates a muscle memory of best practice saving from some future issues.
And one-liner PRs are not bad, they're actually awesome as more people could be involved 🎉

Reply:
Yea, fair enough. I'm going to remove this and move it to it's own PR.